examples/example_annubes.ipynb

```diff
@@ -60,18 +116,59 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "env = AnnubesEnv()\n",
-    "env = DummyVecEnv([lambda: env])\n",
-    "# plot example trials with random agent\n",
-    "data = ngym.utils.plot_env(env, num_trials=5, ob_traces=[\"fixation\", \"start\", \"v\", \"a\"], model=model)"
+    "env1 = AnnubesEnv({\"v\": 1, \"a\": 0})\n",
+    "env1_vec = DummyVecEnv([lambda: env1])\n",
+    "model = A2C(\"MlpPolicy\", env1_vec, verbose=0)\n",
+    "model.learn(total_timesteps=10000)\n",
+    "env1_vec.close()\n",
+    "\n",
+    "# plot example trials with trained agent\n",
+    "data = ngym.utils.plot_env(env1, num_trials=10, ob_traces=[\"fixation\", \"start\", \"v\", \"a\"], model=model)"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "# switch to the second environment and continue training\n",
+    "env2 = AnnubesEnv({\"v\": 0, \"a\": 1})\n",
+    "env2_vec = DummyVecEnv([lambda: env2])\n",
+    "model.set_env(env2_vec)\n",
```

Comment:
Where is it specified that we carrying on from env1? Or, conversely, which line from the `env1` is omitted to ensure that we are not restarting in a fresh env?
I think it would be nice to add a comment here (and perhaps in the `env1` cell) clarifying which (absence of) lines ensure that we are carrying on from the previous env/creating a new env.

Reply:
The model is not created again, it's only the env that changes (`model.set_env`). So the same model keeps getting trained, just with a different env. I'll add a comment for clarifying this.